docs/connector-development/tutorials/cdk-tutorial-python-http/6-read-data.md

```diff
@@ -60,8 +61,8 @@ class ExchangeRates(HttpStream):
             stream_slice: Mapping[str, Any] = None,
             next_page_token: Mapping[str, Any] = None,
     ) -> MutableMapping[str, Any]:
-        # The api requires that we include the base currency as a query param so we do that in this method
-        return {'base': self.base}
+        # The api requires that we include access_key as a query param so we do that in this method
+        return {'access_key': self.access_key}
```

Comment:
Free tier users no longer have the ability to specify a base. Leaving it in results in 400 errors. Although its confusing from a tutorial standpoint to allow for base, base is used in earlier sections to demonstrate invalid configs so I think we should not completely remove it from the tutorial since it still serves a purpose, even if not in the outbound req